RecoLocalMuon/RPCRecHit/plugins/CSCSegtoRPC.cc

```diff
@@ -180,7 +189,6 @@ std::unique_ptr<RPCRecHitCollection> CSCSegtoRPC::thePoints(const CSCSegmentColl
                 RPCGeomServ rpcsrv(rpcId);
 
                 if (dr > 200. || fabs(dz) > 55. || dfg > 1.) {
-                  //if(rpcRegion==1&&dfg>1.&&dr>100.){
```

Comment:
remove

Reply:
Already removed all lines of code that were commented out.